.github/workflows/release.yml

```diff
@@ -0,0 +1,172 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+
+jobs:
+  crates-io:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Push to crates.io
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        run: cargo publish || true
```

Comment:
Why is the `|| true` necessary? Doesn't that mean that the step succeeds even when publishing fails?

Reply:
publishing is disabled until https://github.com/PyO3/pyo3/pull/2786 is merged (or we find some other workaround)

Reply:
i've renamed it to `cargo-publish-dry-run` and passed the dry run flag to publish to better reflect what's actually happening